src/modules/Dashboard/Dashboard.tsx

```
@@ -85,10 +87,17 @@ class Dashboard extends React.Component<Props, States> {
                     debtRegistryEntry,
                 )) as any;
                 const repaymentSchedule = await adapter.getRepaymentSchedule(debtRegistryEntry);
+                const termEndTimestamp = repaymentSchedule.length
+                    ? repaymentSchedule[repaymentSchedule.length - 1]
+                    : moment().unix();
```

I don't think we can assume that the termEndTimestamp is the current time if we don't have a repayment schedule; in what scenarios would we not expect a repaymentSchedule to exist?

I think we should instead throw an error if we don't receive a repaymentSchedule.